utils/raspberrypi/ctt/colors.py

```diff
@@ -0,0 +1,33 @@
+# SIMPLE CODE TO CONVERT RGB VALUES TO LAB
+def RGB_to_LAB(RGB):  # where RGB is a 1x3 array.   e.g RGB = [100, 255, 230]
+    num = 0
+    XYZ = [0, 0, 0]
+    # converted all the three R, G, B to X, Y, Z
+    X = RGB[0] * 0.4124 + RGB[1] * 0.3576 + RGB[2] * 0.1805
+    Y = RGB[0] * 0.2126 + RGB[1] * 0.7152 + RGB[2] * 0.0722
+    Z = RGB[0] * 0.0193 + RGB[1] * 0.1192 + RGB[2] * 0.9505
+    # rounded off the values upto 4 decimal digit
```

Comment:
Was this where we removed the code that was rounding the numbers? Maybe remove the comment if it isn't relevant any more.